### PR TITLE
Fix link in Natss readme pointing to a Knative Setup link which no longer worked

### PR DESCRIPTION
**Pull request by HansK-p**
…

Fixes broken link to Knative Eventing Setup in Natss config readme file.

## Proposed Changes

  * Update Natss config readme link to Knative Eventing Setup

**Release Note**

```release-note
```

### CODE REVIEW

**Comment by n3wscott** [comment]
Need to fix the CLA.

**Comment by HansK-p** [comment]
The cla has hopefully been fixed now as I've added by GitHub username to the signed cla/google. I suppose we'll just give the system some time to find out if it's sufficient/I've done this correctly.

**Comment by HansK-p** [comment]
It doesn't look like the automation is able to detect that I've signed the CLA and that I've also made sure that my GitHub username is now associated with the signed Google Individual CLA agreement.

I'll probably just cancel this PR instead of trying to find out why as this PR is so small that it's probably not worth the hassle (I might try to create another PR later).

Anyway.. I'll wait a couple of more days just in case the automation is able to pick up that I have singed the Google Individual CLA and associated it with my GitHub username (which I assume ought to be sufficient???).

File: natss/config/README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ They do not offer:
 ## Deployment steps
 
 1. Setup
-   [Knative Eventing](https://knative.dev/eventing/blob/master/DEVELOPMENT.md).
+   [Knative Eventing](https://knative.dev/docs/eventing/getting-started/).
```
Comment (aslom) [inline natss/config/README.md]: Correct link is https://github.com/knative/eventing-contrib/blob/master/DEVELOPMENT.md ?

Change to 
```
   [Knative Eventing](https://github.com/knative/eventing-contrib/blob/master/DEVELOPMENT.md).
```

Comment (HansK-p) [reply natss/config/README.md]: I quite honestly kind of forgot the PR. I'll just close the pull request for now.

I'll fight the CLA thing at a later time if I have something a bit more valuable in a PR.
For now I assume (hope) the link will be fixed anyhow.

**Comment by HansK-p** [comment]
I'll might fight the CLA thing at a later time --- and/or come back with a new PR if the link hasn't been fixed the next time I have a need to read the documentation.

**Comment by aslom** [comment]
@HansK-p I hope link will be fixed https://github.com/knative/eventing-contrib/pull/988 and I look forward to work with you on Knative

**Pull request closed without merging**